packages/web-components/src/components/va-alert/va-alert.css

```diff
@@ -166,3 +166,7 @@ i {
 i.fa-times-circle::before {
   content: '\F057'; /* fa-times-circle*/
 }
+
+:host([closeable]:not([closeable='false'])) .alert {
+  padding-right: 5.7rem;
```

Comment:
This ends up being 16px away from the close icon.

Reply:
@humancompanion-usds seemed [interested in having the padding be equal on both sides](https://github.com/department-of-veterans-affairs/component-library/pull/429#discussion_r875850031). Is that still something that we want?

Reply:
We can try it, but I worry that we're going to end up adding back the [problem we were trying to solve](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/658) with that earlier PR. If we add too much padding, alerts with a lot of text (which maybe shouldn't be a thing, but it is) get really long on smaller screens.

Reply:
This padding only takes effect if there's a close icon and it maintains the same padding on both sides relative to the icons. The `Dismissable Background Only Icon` left icon is 20px from the edge. I could update that icon to be 16px from the edge if we want.